kong/db/dao/snis.lua

```diff
@@ -129,7 +154,12 @@ end
 
 -- invalidates the *old* name when updating it to a new name
 function _SNIs:update(pk, entity, options)
-  local _, err, err_t = invalidate_cache(self, self:select(pk))
+  local ok, err, err_t = check_private_key_exists(self, entity)
+  if not ok then
+    return nil, err, err_t
+  end
+
+  _, err, err_t = invalidate_cache(self, self:select(pk))
```

Comment:
`local _` missing, breaks lint

Reply:
I always thought _ had special meaning on its own, until today... Thanks for pointing it out!